### PR TITLE
S4 Header and footer

### PR DESCRIPTION
**Pull request by Zeroya**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -1,7 +1,69 @@
...
-  return <div>Header</div>;
+  const { Option } = Select;
+
+  return (
+    <div className={s.header}>
+      <div className={s.infoBlock}>
+        <div className={s.infoBlock_contacts}>
+          <p>Chat with us</p>
+          <p>+420 336 775 664</p>
+          <p>info@freshnesecom.com</p>
```
Comment (Yuliia143) [inline src/components/Header/Header.tsx]: I don't think that phone number and email should be as p

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -1,7 +1,69 @@
...
+        </div>
+        <div className={s.infoBlock_links}>
+          <p>Blog</p>
+          <p>About Us</p>
+          <p>Careers</p>
```
Comment (Yuliia143) [inline src/components/Header/Header.tsx]: Seems there should be links

File: src/components/Header/Header.tsx [hunk 3]
```diff
@@ -1,7 +1,69 @@
...
+              style={{ width: "50%" }}
+              suffix={<SearchOutlined />}
+              className={s.searchBlock_input_search}
+              placeholder="Search Products, categories ..."
+            />
+          </Input.Group>
+        </div>
+        <div className={s.searchBlock_icons}>
+          <img src={human} />
+          <ShoppingCartOutlined style={{ fontSize: "30px", paddingTop: "8px" }} />
```
Comment (Yuliia143) [inline src/components/Header/Header.tsx]: inline styles

File: src/components/Header/Header.tsx [hunk 4]
```diff
@@ -1,7 +1,69 @@
...
+        </div>
+      </div>
+      <div className={s.selectBlock}>
+        {selectArr.map((el) => {
+          return (
```
Comment (Yuliia143) [inline src/components/Header/Header.tsx]: skip return

File: src/constants/constants.ts [hunk 5]
```diff
@@ -1,2 +1,44 @@
...
+export const infoArr: Array<{ header: string; text: Array<string> }> = [
+  { header: "Get in touch", text: ["About Us", "Careers", "Press Releases", "Blog"] },
+  { header: "Connections", text: ["Facebook", "Twitter", "Instagram", "Youtube", "LinkedIn"] },
+  { header: "Earnings", text: ["Become an Affiliate", "Advertise your product", "Sell on Market"] },
+  {
+    header: "Account",
+    text: ["Your account", "Returns Centre", "100 % purchase protection", "Chat with us", "Help"],
+  },
+];
+export const tagsArr: Array<string> = [
```
Comment (Yuliia143) [inline src/constants/constants.ts]: Seems it's mocked data, should be in another file

File: src/components/Footer/Footer.module.scss [hunk 6]
```diff
@@ -0,0 +1,43 @@
+.footer {
+  margin: 64px 0 64px 0;
+
+  .infoBlock {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+
+    &_section p {
+      color: #6A983C;
```
Comment (Yuliia143) [inline src/components/Footer/Footer.module.scss]: Create file for colors

File: src/components/Footer/Footer.module.scss [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+      align-items: center;
+      flex-wrap: wrap;
+      gap: 10px;
+
+      &>div {
+        background: #F5F5F5;
+        border-radius: 12px;
+        padding: 4px 10px;
+
+        &>p {
```
Comment (Yuliia143) [inline src/components/Footer/Footer.module.scss]: Don't like these selectors, try to create classes more

File: src/components/Footer/Footer.tsx [hunk 8]
```diff
@@ -1,7 +1,39 @@
...
+              ))}
+            </div>
+          );
+        })}
+      </div>
+      <div className={s.tagsBlock}>
+        <h2>Product tags</h2>
+        <div className={s.tagsBlock_list}>
+          {tagsArr.map((tag) => {
+            return (
```
Comment (Yuliia143) [inline src/components/Footer/Footer.tsx]: Follow one code style

File: src/components/Header/Header.tsx (hunk 1 above)
Comment (Zeroya) [reply src/components/Header/Header.tsx]: Fixed

File: src/components/Header/Header.tsx (hunk 2 above)
Comment (Zeroya) [reply src/components/Header/Header.tsx]: In general, we don't have such pages, but I posted a link to existing ones

File: src/components/Header/Header.tsx (hunk 4 above)
Comment (Zeroya) [reply src/components/Header/Header.tsx]: Fixed

File: src/components/Header/Header.tsx (hunk 3 above)
Comment (Zeroya) [reply src/components/Header/Header.tsx]: moved to scss file

File: src/constants/constants.ts (hunk 5 above)
Comment (Zeroya) [reply src/constants/constants.ts]: Created a folder for mocked data

File: src/components/Footer/Footer.tsx (hunk 8 above)
Comment (Zeroya) [reply src/components/Footer/Footer.tsx]: You mean return in map ?

Comment (Yuliia143) [reply src/components/Footer/Footer.tsx]: Yep
Sometimes you have it, sometimes don't

Comment (Zeroya) [reply src/components/Footer/Footer.tsx]: I already changed it

**Review by Yuliia143 — CHANGES_REQUESTED** [review]
:)

File: src/components/Header/Header.tsx (hunk 3 above)
Comment (Zeroya) [reply src/components/Header/Header.tsx]: I think i got it, its inline styles about which I forgot , correct ?

Comment (Yuliia143) [reply src/components/Header/Header.tsx]: You're right

**Review by Yuliia143 — APPROVED** [review]

**Pull request merged**